content/collections/flutter_sdk/en/flutter-sdk-4-0.md

```diff
@@ -68,44 +92,68 @@ class YourClass {
+| ----------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------- |
+| `defaultTracking`             | `DefaultTrackingOptions`. Options to control the default events tracking.                                                                                                                                                                                                                                                | Check [Tracking default events](#tracking-default-events). |
+| `enableCoppaControl`          | `bool`. Whether to enable COPPA control for tracking options.                                                                                                                                                                                                                                                            | `false`                                                    |
+| `flushEventsOnClose`          | `bool`. Flush unsent events on app close.                                                                                                                                                                                                                                                                                | `true`                                                     |
+| `identifyBatchIntervalMillis` | `int`. The amount of time SDK attempts to batch intercepted identify events. The value is in milliseconds                                                                                                                                                                                                                | `30000`                                                    |
+| `migrateLegacyData`           | `bool`. Whether to migrate maintenance Android SDK and maintenance iOS SDK data (events, user/device ID). Learn more at the configuration section of the underlying [Kotlin SDK](/docs/sdks/analytics/android/android-kotlin-sdk/#configuration) and [Swift SDK](/docs/sdks/analytics/ios/ios-swift-sdk/#configuration). | `true`                                                     |
+
+### Configuration for web and Android
+
+| Name       | Description                                                                                                 | Default Value |
+| ---------- | ----------------------------------------------------------------------------------------------------------- | ------------- |
+| `deviceId` | `String`. The device ID to use for this device. If you don't provide a `deviceId`, Amplitude generates one. | `null`        |
```

Comment:
Surprised that android sdk supports device id https://github.com/amplitude/Amplitude-Kotlin/blob/74764561d2c87c5eb6f59198822f93ca249178ad/core/src/main/java/com/amplitude/core/Configuration.kt#L34 but the ios sdk doesn't https://github.com/amplitude/Amplitude-Swift/blob/98a5f020359d6e4c02f774a57d31b2fc7d062f46/Sources/Amplitude/Configuration.swift#L185

Reply:
@crleona @igor-amp any ideas?